builder/builder-next/controller.go

```diff
@@ -144,6 +144,11 @@ func newController(rt http.RoundTripper, opt Opt) (*control.Controller, error) {
 		return nil, errors.Wrap(err, "could not get builder GC policy")
 	}
 
+	ents, err := getEntitlements(opt.BuilderConfig)
+	if err != nil {
```

Comment:
getEntitlements never returns an error, might as well inline it into the struct below:
```
Entitlements: getEntitlements(opt.BuilderConfig),
```